recipes/protobuf/all/test_v1_package/addressbook.proto

```diff
@@ -0,0 +1,51 @@
+// See README.txt for information and build instructions.
```

Comment:
test v1 package should reuse .cpp & .proto files of test package to avoid code duplication